hoot-services/src/main/java/hoot/services/language/SupportedLanguage.java

```diff
@@ -50,4 +52,35 @@ public SupportedLanguage() {}
 
   public boolean getAvailable() { return available; }
   public void setAvailable(boolean available) { this.available = available; }
```

Comment:
There were issues here where LanguageResource expected a copy of this class in order to modify its data but was getting a ref by accident...so fixed.